assets/component-cart-items.css

```diff
@@ -156,7 +156,7 @@ cart-remove-button .icon-remove {
   }
 }
```

Comment:
In the filter, when the body is set to the highest size it's cutting the text in some languages (screenshot in French):

https://screenshot.click/05-22-khmva-ekybx.png

Also the focus is cutting of the text next to it: https://screenshot.click/05-23-0npqv-8vhyv.mp4